spec/bowling_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'bowling'
+
```

Comment:
Your testing is generally really thorough and clear as to what you're testing for.